modules/ROOT/pages/functions/mathematical-logarithmic.adoc

```diff
@@ -89,6 +89,7 @@ e(expression)
 |===
 
 | `exp(null)` returns `null`.
+| `exp()` returns `Infinity` when the return value is greater than the largest `FLOAT` value.
```

Comment:
If the docs reviewer agrees, maybe we should add a note here to what the largest FLOAT values is (Java double.MAX_VALUE I believe)